database/migrations/20171212141652_feed.js

```diff
@@ -0,0 +1,18 @@
+exports.up = (knex, Promise) => {
+  return Promise.all([
+    Promise.resolve(knex.schema.dropTableIfExists('feed_indices')),
+    knex.schema.createTable('feed_indices', (table) => {
+      table.increments('id').primary();
+      table.integer('user_id');
```

Comment:
Suggestion: Add ".unique()" for the user_id field since you only want to have one row per user